feat: add examples using split card elements (#50)
Basis-Theory/basis-theory-js-examples: collect-cards-with-elements-react/README.md

> This example shows how to securely collect card information with [Basis Theory React](https://github.com/Basis-Theory/basis-theory-react).

Reviewer: ```suggestion
This example shows how to securely collect card information with [Basis Theory React](https://github.com/Basis-Theory/basis-theory-react) using the full featured `CardElement`.
```

> In this example, the the single `CardElement` is used.

Reviewer: ```suggestion
```

> For an example using the split card elements (`CardNumberElement`, `CardExpirationDateElement`, `CardVerificationCodeElement`), check the code at [this folder](https://github.com/Basis-Theory/basis-theory-js-examples/tree/master/collect-cards-with-split-elements-react).

Reviewer: ```suggestion
Alternatively, head over [here](https://github.com/Basis-Theory/basis-theory-js-examples/tree/master/collect-cards-with-split-elements-react) to check out how to do the same thing but using `CardNumberElement`, `CardExpirationDateElement`, `CardVerificationCodeElement`.
```
